RWKV-v4neo/src/model.py

```diff
@@ -151,21 +151,20 @@ def RUN_CUDA(B, T, C, w, u, k, v):
 
 
 class LoraLinear(nn.Module):
-
     def __init__(self, in_features: int, out_features: int, bias: bool):
         super().__init__()
 
-        self.weight = nn.Parameter(torch.empty((out_features, in_features)))
+        self.weight = nn.Parameter(torch.empty((out_features, in_features), dtype=DTYPE), requires_grad=False)
```

Comment:
If I remember it correctly, the `dtype` is globally specified in the Lightning wrapper. Should not need to give it here explicitly.